src/interfaces/api-filing-interface.ts

```diff
@@ -67,7 +67,11 @@ export interface ApiFilingIF {
     }
 
     // dissolution filings only
-    dissolution?: any // some object
+    dissolution?: {
+      custodialOffice?: any // FUTURE: use a proper address type here
```

Comment:
The [schema](https://github.com/bcgov/business-schemas/blob/8453db2457e4324449cde3d7bc66887441972045/src/registry_schemas/schemas/dissolution.json#L54) says this is an [office type](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/office.json) but I'd like to see an example before making the change here, so I'm leaving it as `any` for now.